sphinx/builders/latex/theming.py

```diff
@@ -24,61 +24,89 @@
 class Theme:
     """A set of LaTeX configurations."""
 
+    LATEX_ELEMENTS_KEYS = ['papersize', 'pointsize']
+    UPDATABLE_KEYS = ['papersize', 'pointsize', 'show_table_column_rules']
+
     def __init__(self, name: str) -> None:
         self.name = name
         self.docclass = name
         self.wrapperclass = name
+        self.papersize = 'letterpaper'
+        self.pointsize = '10pt'
+        self.show_table_column_rules = True
```

Comment:
Now I use a boolean to show/hide column rules. But it might be better to choose a character for column separator instead.
```
table_column_rules = |
```
```
table_column_rules = ||
```
```
table_column_rules = :
# see https://tex.stackexchange.com/questions/122301/how-to-add-vertical-and-horizontal-dashed-lines-in-array-and-tabular-enviroments
```

Reply:
In addition, we can use `:stub-columns:` option to make the table having vertical headings. So special column rule; header separator would be needed for such case...